### DISCUSSION TITLE
Battery Health No data??

### DISCUSSION THREAD

**Discussion by ArvinLiYongJun** [teslamate-org/teslamate#3569] [Q&A] [answered]
I started using Teslamate yesterday but found that the battery health dashboard has no data. Is this normal? Or do I need to use it for a while?
![Snipaste_2023-12-15_13-04-48](https://github.com/teslamate-org/teslamate/assets/42535646/bf925fc5-e600-4be1-aa6c-32cd4ec987c1)

**Comment by ArvinLiYongJun** [comment]
I saw a red exclamation mark in the error report, and I was worried that there was something wrong?
![Snipaste_2023-12-15_13-08-39](https://github.com/teslamate-org/teslamate/assets/42535646/bbfd0c76-8e8d-461c-96d5-5feb4b2df3dd)

**Reply by cwanja** [reply]
Totally normal. Review https://docs.teslamate.org/docs/faq#why-are-no-consumption-values-displayed-in-grafana

**Reply by cwanja** [reply]
Are your Drives and Charges dashboards working and collecting data?

**Comment by ArvinLiYongJun** [comment]
Yes,it's working.





------------------ Original ------------------
From: Chris Wanja ***@***.***&gt;
Date: Fri,Dec 15,2023 11:53 PM
To: teslamate-org/teslamate ***@***.***&gt;
Cc: Arvin ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [teslamate-org/teslamate] Battery Health No data?? (Discussion#3569)





 
Are your Drives and Charges dashboards working and collecting data?

**Reply by cwanja** [reply, marked as answer]
TeslaMate is correctly collecting data. Please review the [FAQ](https://docs.teslamate.org/docs/faq#why-are-no-consumption-values-displayed-in-grafana) link above regarding consumption values. Specifically:
>  It takes at least two charging sessions before the first estimate can be displayed. Charging sessions have to be longer than 10 minutes and less than 95% state-of-charge (SoC). Each future charging session will slightly improve the accuracy of the estimate, which is applied retroactively to all data.

**Reply by cwanja** [reply]
@jheredianet can confirm that the Battery Health dashboards require efficiency data, which requires the charing sessions previously mentioned.

**Reply by ArvinLiYongJun** [reply]
Thank you for your answer. I will charge it twice and see if the data can be displayed normally.

**Reply by ArvinLiYongJun** [reply]
Also, I would like to ask why there is no data for Location?

**Reply by cwanja** [reply]
> Also, I would like to ask why there is no data for Location?

How many drives have you completed? Have you checked your `addresses` table?

**Reply by ArvinLiYongJun** [reply]
It's empty
![Uploading Snipaste_2023-12-19_18-29-04.png…]()

**Reply by cwanja** [reply]
> It's empty
> 
> ![Uploading Snipaste_2023-12-19_18-29-04.png…]()
> 
> 
> 
> 

Image never came through. Have you completed any drives? Share what that dashboard shows.

**Reply by ArvinLiYongJun** [reply]
![Snipaste_2023-12-19_18-29-04](https://github.com/teslamate-org/teslamate/assets/42535646/d6afe8a3-e2f3-4830-9f34-509d7756c240)
![Snipaste_2023-12-20_09-59-59](https://github.com/teslamate-org/teslamate/assets/42535646/39bbedf3-7cfa-4d82-a101-84de9dbc3883)

Now, should be able to see it

**Reply by cwanja** [reply]
What does TeslaMate logs report? Drives are not fetching the addresses which is why your Locations dashboard is empty.

**Reply by ArvinLiYongJun** [reply]
I found that my DRIVES did not obtain address information. What is the reason? That's why my LOCATIONS is empty. Where can I see the log? I'm not an IT person, I may need detailed steps. Thanks.
